connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -750,4 +755,41 @@ private String trace(Throwable t) {
         return keys;
     }
 
+    @Override
+    public List<ConfigKeyInfo> connectorPluginConfig(String pluginName) {
```

Comment:
If we continue to put the onus on the `Herder` to ensure that the requested plugin has a supported type (e.g., throw an error if it's a REST extension), then I think "connector plugin" is useful since it differentiates between worker and connector plugins.

But it looks like the `ConnectorPluginInfo` class is itself completely agnostic on the connector-plugin vs. worker-plugin front, and it might even possible to leverage it with no modifications if we eventually decide to add support for exposing worker plugin information to the REST API. So I think renaming that to `PluginInfo` (or something like that) would be reasonable.